packages/oui-navbar/README.md

```diff
@@ -0,0 +1,273 @@
+# Navbar
```

Comment:
We will have to find a solution to get a more readable documentation for complex components like this one!